src/lsst_efd_client/efd_helper.py

```diff
@@ -588,6 +597,25 @@ async def select_packed_time_series(
         vals.update({"times": df["times"]})
         return pd.DataFrame(vals, index=df.index)
 
+    async def _is_topic_valid(self, topic: str) -> bool:
+        """Check if the specified topic is in the schema.
+        A topic is valid and returns ``True`` if it is in the cached list of
+        topics. Any other case returns ``False``.
```

Comment:
Single backticks can be used for Python core constants.
```suggestion
        A topic is valid and returns `True` if it is in the cached list of
        topics. Any other case returns `False`.
```

Reply:
solved